src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+const MALE_SIGN = 'm';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age
+      && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )
+    }
+
+    <p className="Person__partner">
+      {!person.isMarried && ('I am not married')}
+      {person.isMarried && (
+        person.sex === MALE_SIGN ? (
+          `${person.partnerName} is my wife`
+        ) : (
+          `${person.partnerName} is my husband`
+        ))}
```

Comment:
it can be replace with ternary like 
```jsx
            {person.isMarried
              ? ...
              : ...}
```

If you don't want to use ternary inside ternary try to create separate variable